src/assets/content/dictionary/it-works-on-my-machine.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: it works on my machine
-definition: ''
+definition: 'A funny (yet embarrassing) phrase commonly used by programmers or anyone who has written some code to explain an error or bug that might have occurred in production or during deployments and which worked perfectly fine during development'
```

Comment:
haha! love this. 
nitpick: the definition is missing a period at the end

Reply:
fixed✅